src/PureScript/Backend/Chez/Convert.purs

```diff
@@ -150,6 +150,17 @@ codegenTopLevelBinding codegenEnv (Tuple (Ident i) n) =
 
 codegenExpr :: CodegenEnv -> NeutralExpr -> ChezExpr
 codegenExpr codegenEnv@{ currentModule } s = case unwrap s of
+  Var (Qualified (Just (ModuleName "Data.List.Types")) (Ident "Cons")) ->
+    -- (lambda (x) (lambda (xs) (scm:cons x xs)))
+    S.List
+      [ S.Identifier (scmPrefixed "lambda")
+      , S.List [ S.Identifier "x" ]
+      , S.List
+          [ S.Identifier $ scmPrefixed "lambda"
+          , S.List [ S.Identifier "xs" ]
+          , S.List [ S.Identifier $ scmPrefixed "cons", S.Identifier "x", S.Identifier "xs" ]
+          ]
+      ]
```

Comment:
We might want to add this to the runtime module.

Reply:
Why might we want to add that?

Reply:
I made this change. I think it's slightly better.